packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/lib/index.js

```diff
@@ -80,7 +80,7 @@ let report = function(event, context, responseStatus, physicalResourceId, respon
 const requestCertificate = async function(requestId, domainName, subjectAlternativeNames, hostedZoneId, region) {
   const crypto = require('crypto');
   const acm = new aws.ACM({ region });
-  const route53 = new aws.Route53();
+  const route53 = region.startsWith("cn-") ? new aws.Route53({endpoint: "https://route53.amazonaws.com.cn"}) : new aws.Route53();
```

Comment:
* what're the exact region(s)? is there a good reason for going with `startsWith`? (i.e. would all future CN regions map to the same endpoint? if not, we should be specific here)
* how has this been tested? - I don't see Route53 as a [supported service](https://www.amazonaws.cn/en/about-aws/regional-product-services/) in the region table.
  * can you share any resources used in determining this endpoint?